test/int/adapter.test.ts

```diff
@@ -85,5 +89,52 @@ suite('Chrome Debug Adapter etc', function () {
                 dc.assertStoppedLocation('debugger_statement', { path: breakFile, line: DEBUGGER_LINE } )
             ]);
         });
+
+        test('Should attach to existing instance of chrome and break on debugger statement', async () => {
+            const testProjectRoot = path.join(DATA_ROOT, 'intervalDebugger');
+            const breakFile = path.join(testProjectRoot, 'src/app.ts');
+            const DEBUGGER_LINE = 2;
+
+            server = createServer({ root: testProjectRoot });
+            server.listen(7890);
```

Comment:
This code is similar to line 118 and to the should stop on debugger statement in http://localhost test
Is there any way we can avoid duplicating it?

Reply:
Addressed. We set this up in the `setup()` for this suite now